data/afoxbr

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Antony",
+  "url": "https://github.com/afoxbr",
+  "image": "https://avatars.githubusercontent.com/u/106445119?s=400&u=74f98a61140e7086cbd799501d524b83b189224e&v=4",
+  "why": "I'd like to meet other devs who are into fitness. I'm new to tech and returning to fitness after a long time away.",
+  "weekly": [lift x3, run x5],
```

Comment:
Kindly make each weekly goal as a string enclosed in " ".
So, code to be updated:
"weekly": ["lift x3", "run x5"],

Reply:
Done. Apologies, should've spotted that before commit.

Reply:
No problem, btw thanks for your contribution